test/cpp/interop/xds_interop_client.cc

```diff
@@ -114,6 +132,7 @@ class XdsStatsWatcher {
   int rpcs_needed_;
   int no_remote_peer_ = 0;
   std::map<std::string, int> rpcs_by_peer_;
+  std::map<std::string, std::map<std::string, int>> rpcs_by_type_;
```

Comment:
Please add inline comments indicating that the top-level key is the RPC method and the lower-level key is the peer name.